### PR TITLE
`Development`: Update programming exercise grading documentation

### PR DESCRIPTION
**Pull request by Strohgelaender**
#### General
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/#naming-conventions-for-github-pull-requests).

### Motivation and Context
Keep the documentation up to date

### Description
- Update images
- Re-order setup information to the new order used in Artemis
- Update grading documentation to explain task-based grading
- Add a screencast explaining the grading confirguration
- Simplify structure

### Steps for Testing
view the documentation: https://artemis-platform--7511.org.readthedocs.build/en/7511/

#### Code Review
- [x] Code Review 1
- [x] Code Review 2

### Screenshots

old structure:
grafik

new structure: 
grafik

### CODE REVIEW

**Review by tobias-lippert — CHANGES_REQUESTED** [review]
Please do not use test users in your videos. Either cut out the navbar post production or use a TUM Login.

**Review by tobias-lippert — DISMISSED** [review]
LGTM now

**Review by xHadie — CHANGES_REQUESTED** [review]
Overall pretty good and understandable, but there are some grammar mistakes in your changes. I would suggest putting your changes through a grammar checker to polish things up a bit!

File: docs/user/exercises/programming-exercise-setup.inc [hunk 1]
```diff
@@ -514,25 +515,37 @@ Exercise Creation
...
     Artemis always grades new submissions with the latest configuration but existing submissions might have been graded with an outdated configuration. Artemis warns instructors about grading inconsistencies with the **Updated grading** badge.
 
-- **Test Case Tab**: Adapt the contribution of each test case to the overall score
+- **Test Case Tab**: Adapt the contribution of each test case to the overall score or set the grading based on entire tasks.
+
+To get a better understanding of the feature, you can watch the following video:
+
+  .. raw:: html
+
+    <iframe src="https://tum.live/w/artemisintro/40748?video_only=1&t=0" allowfullscreen="1" frameborder="0" width="600" height="400">
```
Comment (simon-hng) [inline docs/user/exercises/programming-exercise-setup.inc:524]:
```suggestion
    <iframe src="https://tum.live/w/artemisintro/40748/PRES?video_only=1&" allowfullscreen="1" frameborder="0" width="600" height="400">
```

**Review by tobias-lippert — DISMISSED** [review]
Reapprove

**Review by xHadie — APPROVED** [review]
LGTM 🙂

**Review by tobias-lippert — DISMISSED** [review]
Reapprove

File: docs/user/exercises/programming-exercise-setup.inc [hunk 2]
```diff
@@ -1,111 +1,24 @@
...
-Exercise Creation
-^^^^^^^^^^^^^^^^^
+Creating an exercise consists of the following steps:
 
-1. **Open Course Management**
+#. `Generate programming exercise`_: Start by configuring basic settings and add a new exercise to the course.
+#. `Update exercise code in repositories`_: Update the template, solution, and test-repositories to contain the code of your exercise.
+#. `Adapt the build plans`_ (optional): If needed, tailor the build plans to suit your requirements.
+#. `Configure static code analysis`_ (optional): If static code analysis is activated, customize the configuration as per your needs.
+#. `Adapt the interactive problem statement`_: Clearly define your exercise in the problem statement Use tasks to create a group of test cases.
```
Comment (b-fein) [inline docs/user/exercises/programming-exercise-setup.inc:9]:
```suggestion
#. `Adapt the interactive problem statement`_: Clearly define your exercise in the problem statement. Use tasks to create a group of test cases.
```

**Review by b-fein — APPROVED** [review]

**Review by b-fein — APPROVED** [review]

**Review by tobias-lippert — APPROVED** [review]
Reapprove

File: docs/user/exercises/programming-exercise-import.inc [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    .. figure:: programming/course-management-exercise-dashboard-import.png
+              :align: center
+
+- Select an exercise to import
+
+    .. figure:: programming/course-management-exercise-dashboard-import-modal.png
+              :align: center
+
+.. note::
+  Instructors can import exercises from courses, where they are registered as instructors
```
Comment (LenaKahle) [inline docs/user/exercises/programming-exercise-import.inc:24]: unnecessary comma (from courses where they are registered)

**Review by LenaKahle — APPROVED** [review]
Looks good, I added 2 comments for small grammar/typo issues.

**Review by tobias-lippert — APPROVED** [review]
Reapprove

**Review by b-fein — APPROVED** [review]

**Review by JanaNF — APPROVED** [review]
Looks good

**Review by LenaKahle — APPROVED** [review]
Reapprove.

**Pull request merged**